### ISSUE TITLE
Tests are 404ing

### ISSUE THREAD

**Issue by kiliankoe** [kollemate/kolledata#62]
Why is this? They worked (partially) at least for a little while when they were written, but now every single one of them awaiting a 200 HTTP status is failing. The ones expecting redirects still work, so it can't be broken routes.

Trying to GET the root route for example:

```
2) GET some pages GET / should return 200 OK:
     Error: expected 200 "OK", got 404 "Not Found"
      at Test.assert (/Users/kilian/develop/kolledata/node_modules/supertest/lib/test.js:205:15)
      at Server.assert (/Users/kilian/develop/kolledata/node_modules/supertest/lib/test.js:132:12)
      at Server.g (events.js:180:16)
      at Server.emit (events.js:92:17)
      at net.js:1276:10
      at process._tickCallback (node.js:419:13)
```

Calling the site itself through a browser returns the correct codes, so it really should be the tests acting up instead of the error handling.

(Run `npm test` to run the tests through mocha btw)

**Comment by lucaswo** [comment]
At least it isn't the travis foo anymore, causing the tests to fail. (See 57c3b010c573cacb9db09845b55d7600f052609a)

**Comment by kiliankoe** [comment]
This error was caused by the tests failing to send a request header which the langController required. You and Patrick fixed this in fc69560e6d383c355cfe013e3c5d9881e6f2420c. 

The other errors were caused by other stuff (like the `/api` routing no longer actually existing and async being a son of a bitch).
